resources/i18n/en.json

```diff
@@ -121,5 +121,6 @@
     "open-link-in-web-browser":"Open link in web browser",
     "download-dir-dialog-title":"Are you sure you want to change the download directory?",
     "download-dir-dialog-msg":"The new download directory path will be:\n{{DIRECTORY}}",
-    "invalid-port":"Invalid port"
+    "invalid-port":"Invalid port",
+    "zim-open-fail":"The zim file has been modified on disk and cannot be loaded properly.\nPlese undo the changes and re-open it."
```

Comment:
"The ZIM file %1 cannot be open properly. It will be removed from your library."

Reply:
What should be shown at %1? ZIM title or full path?

Reply:
fullpath